lib/ssl/src/ssl_record.hrl

```diff
@@ -180,4 +180,17 @@
           next_iv  % opaque IV[SecurityParameters.record_iv_length];
          }). 
 
+
+-define('TLS-1.X', {3, _}).
+-define('TLS-1.3', {3, 4}).
+-define('TLS-1.2', {3, 3}).
+-define('TLS-1.1', {3, 2}).
+-define('TLS-1.0', {3, 1}).
+-define('DTLS-1.X', {254, _}).
+-define('DTLS-1.0', {254, 255}).
+-define('DTLS-1.1', {254, 254}).
```

Comment:
When I was thinking about this several years ago (when we optimized the distribution over SSL) I was thinking if it could be a good idea to represent the version as an integer. And when we now are declaring the macros there is actually not much point in having the version as a tuple.
```suggestion
-define('is_TLS-1.X'(V), (V bsr 8) == 3). % to be used in guard or case or if statement
-define('TLS-1.3', 16#304).
-define('TLS-1.2', 16#303).
-define('TLS-1.1', 16#302).
-define('TLS-1.0', 16#301).
-define('is_DTLS-1.X'(V), (V bsr 8) == 254. % to be used in guard or case or if statement
-define('DTLS-1.0', (254 bsl 8) + 255.
-define('DTLS-1.1', (254 bsl 8) + 254).
```
There will be a need for minor changes at a few other places as well, for example when matching with TLS-1.X and when encoding /decoding the SSL_record from and to bytes.

Reply:
I don't know if you like this change, but I do.

Reply:
I can see both benefits and drawbacks. I think we should not make this change for this PR but we can evaluate if we think it is beneficial enough later,

Reply:
What are the drawbacks?